### ISSUE TITLE
Release v1.32.8

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#1606]
:robot: A dedicated issue to run tests and deploy release [v1.32.8](https://github.com/deckhouse/deckhouse/milestone/52).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | gcp | azure | openstack | yandex-cloud | vsphere | static`
  - `git_ref` is a tag or branch: `vX.Y.Z | test-X.Y.Z* | main | release-X.Y`
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `containerd | docker`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.19 | 1.20 | 1.21 | 1.22 | 1.23`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag or branch: `vX.Y.Z | test-vX.Y.Z* | main | release-X.Y`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.21'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.19
/e2e/use/k8s/1.21

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.19 and 1.21.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build modules FE' and 'build FE' jobs are finished.

</details>

**Comment by deckhouse-BOaTswain** [comment]
New commit [8aaac6](https://github.com/deckhouse/deckhouse/commit/8aaac617814fceedaed29898c0b3272586b86bec) in branch **main**:
```
Revert "[monitoring-kubernetes] Enable systemd collector in the node-exporter (#768)" (#1609)

This reverts commit d7bbd93d1e6001e8a8aa7582e21df45e94da5830.

Signed-off-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [197a21](https://github.com/deckhouse/deckhouse/commit/197a21af323f478eb85778c70be3360ce09b4c04) in branch **main**:
```
[cloud-provider-azure] type -> skuName (#1598)

Signed-off-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [4f2a53](https://github.com/deckhouse/deckhouse/commit/4f2a532ae0a046cad0887977d84241bb0e855daa) in branch **main**:
```
[kubernetes-api-proxy] prepull image (#1608)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [a219a5](https://github.com/deckhouse/deckhouse/commit/a219a51bb1661ed99cd617fb1276971183e154ce) in branch **main**:
```
[docs] Fix rendering CRD file with dots in the name (#1610)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2dafe7](https://github.com/deckhouse/deckhouse/commit/2dafe7794f987593a8cd53cfd2dcc0126217f975) in branch **main**:
```
[testing] Fixed e2e tests in Yandex.Cloud and Azure (#1597)

Signed-off-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ba586c](https://github.com/deckhouse/deckhouse/commit/ba586cbe2fed5a12aa18866c77079fe7f59a2f4a) in branch **main**:
```
Re-generate changelog (#1607)

Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.32.8** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [d2269c](https://github.com/deckhouse/deckhouse/commit/d2269c12abba2584dd2fe2431d58fa27a7fb3908) in branch **main**:
```
[log-shipper] Provide a structural schema for log-shipper CRD (#1612)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [0b8271](https://github.com/deckhouse/deckhouse/commit/0b8271896a66bb8c299aa03e24bdb8580badb075) in branch **main**:
```
[ci] Add Makefile (#1593)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
Co-authored-by: Andrei Kvapil <kvapss@gmail.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [eca940](https://github.com/deckhouse/deckhouse/commit/eca940c34e23d2a59bb2e7faf0fd84a44462bac4) in branch **main**:
```
[ingress-nginx][docs] Update README of the protobuf-exporter (#1614)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [566a29](https://github.com/deckhouse/deckhouse/commit/566a297794ce5e5c56a02fc31ded7b092891401b) in branch **main**:
```
[site] Fix styles of the ordered list on the site (#1624)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b12615](https://github.com/deckhouse/deckhouse/commit/b12615208f8d8ef1d4f2904825b5e2a389a0db7c) in branch **main**:
```
[candi] fix registry packages install scripts for centos (#1621)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [1e97bf](https://github.com/deckhouse/deckhouse/commit/1e97bf3499c51cd68ed938377974a6bd03f6d830) in branch **main**:
```
[upmeter] Remote write exporter: don't retry sending metrics on 4xx errors from storage (#1579)

Signed-off-by: Eugene Shevchenko <evgeny.shevchenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [dd5549](https://github.com/deckhouse/deckhouse/commit/dd55495f9a14588194d2c4b389befe88f188f75b) in branch **main**:
```
[prometheus] fix: rename errors related metrics (#1627)

- https://github.com/flant/addon-operator/pull/133
Signed-off-by: Ivan Mikheykin <ivan.mikheykin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [119879](https://github.com/deckhouse/deckhouse/commit/119879aa88851643100777e87a24cf9844b44503) in branch **main**:
```
[cert-manager] Added owner ref certificate parameter (#1601)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [093c7e](https://github.com/deckhouse/deckhouse/commit/093c7e47b3a51522c6fcad765d63cf33215870da) in branch **main**:
```
[user-authz] Fix gitlab-runner access rights doc (#1611)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
Co-authored-by: Artem Kladov <artem.kladov@flant.com>
```
